airbyte-webapp/src/components/connectorBuilder/Builder/UiYamlToggleButton.module.scss

```diff
@@ -0,0 +1,36 @@
+@use "scss/variables";
+@use "scss/colors";
+
+.button {
+  cursor: pointer;
+  border: variables.$border-thin solid colors.$dark-blue;
+  background-color: colors.$dark-blue;
+  display: flex;
+  border-radius: 6px;
+  padding: 0;
+  overflow: hidden;
+  height: 21px;
```

Comment:
do we need to hardcode the size like this? It seems like we can instead work with paddings

Reply:
I think I realized why it was done this way - it's to make sure both sides are the same size. You can do this with grid layout:
```
  display: grid; 
  grid-template-columns: 1fr 1fr; 
  grid-template-rows: 1fr; 
  gap: 0px 0px; 
  grid-template-areas: 
    ". ."; 
```

Reply:
That definitely seems like a better way to do it. Thanks for the suggestion!

Reply:
Haven't tried this locally, but shouldn't it be possible to just shorten this to?

```css
  display: grid; 
  grid-template-columns: 1fr 1fr; 
```

Reply:
Seems like that works too, good catch. I need to update my knowledge of CSS grid, it has been a while since I used it :)